notebook/notebookapp.py

```diff
@@ -399,11 +398,6 @@ def _log_format_default(self):
         """override default log format to include time"""
         return u"%(color)s[%(levelname)1.1s %(asctime)s.%(msecs).03d %(name)s]%(end_color)s %(message)s"
 
-    ignore_minified_js = Bool(False,
-            config=True,
-            help='Use minified JS file or not, mainly use during dev to avoid JS recompilation', 
-            )
-
```

Comment:
Keep this, emit a DeprecationWarning + a log warning message that it is deprecated, does not work and will be remove in 6.0, we have some instruction in the wild that say how to use it. 

I'm fine making it even refuse to start for after 5.0 but the during 5.0.dev transition, we can be explicit.